src/SignalRServiceExtension/TriggerBindings/InvocationContextExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Management;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    public static class InvocationContextExtensions
+    {
+        public static async Task<IHubClients> GetClientsAsync(this InvocationContext invocationContext)
```

Comment:
possilble to put Clients/Groups/UserGroups to InvocationContext?
Looks the Clients/Groups/UserGroups are only for C#, maybe add a new output of SignalRTriggerAttri that's only for C#?
Then the interface will be much similar to IHubContext

Reply:
Or add a new class implements IHubContext and use the InvocationContext as ctor param?

Reply:
I think in most C# cases, they should use class based model. Then they use that just like what they do in SDK. And for other few customers, we provided an extension to let them use it easily. It's not a common case, so as simple as this is OK I think.
Put them to InvocationContext is not good, and InvocationContext should keep simple as it passes between multiple language.